versions.json

```diff
@@ -11,14 +11,9 @@
 	    "download_sha256": "194bcc256c187a56a2891c9bdc36f2112bd6035e40a35f877b2f52524a0888ce"
 	},
 	{
-	    "major": "24.2",
-	    "version": "24.2.1",
-	    "download_sha256": "380a77aef34ad449bf8370a380b7901364b9be23e3d25068fc5c43258bcbec11"
-	},
-	{
-	    "major": "25.0",
```

Comment:
You could add 24.3 without removing 25.0-rc1. If there are valid reason for removing 25-rc2, then please state them.

Reply:
`25.0-rc1` is not builded at all in GH Actions, locally all is ok, in GHA we have errors. For save time - I was want to remove non working image building

Reply:
looks like the ARMv8 (aarch64) image failed. Disabling that architecture should be enough to fix the build failure and would speed up the builds.
I still think that having ARM images is a good idea, but so far we don't use them.

Reply:
seems there is a valid use case for Aarch64 images (on Apple M1 systems)...